### ISSUE TITLE
Errored Coronagraph Imagery

### ISSUE THREAD

**Issue by Jesterface23** [Helioviewer-Project/api#186]
Hello, I have a list compiled of images that may need to be reprocessed,
[Imagery List.xlsx](https://github.com/Helioviewer-Project/api/files/9186103/Imagery.List.xlsx)

STEREO A and B C2 imagery has whiteout images. SOHO C3 imagery has a mix of of all sorts of issues compared to different C3 imagery sources.

SOHO C3 imagery is missing for all of 2009/12/31.

STEREO A throws an error for the timestamps between 2022/12/25 06:39Z and 07:24Z
org.helioviewer.jhv.io.LoadLayer.request - {"frames":5,"message":null,"uri":"jpip://helioviewer.ias.u-psud.fr:8080//movies/STEREO_A_SECCHI_COR2_white-light_F2011-12-25T06.31.30Z_T2011-12-25T07.31.30ZL.jpx"}
org.helioviewer.jhv.io.LoadLayer$Callback.onFailure: Duplicated time stamps: jpip://helioviewer.ias.u-psud.fr:8080//movies/STEREO_A_SECCHI_COR2_white-light_F2011-12-25T06.31.30Z_T2011-12-25T07.31.30ZL.jpx

Also to note, for STEREO A from epoch 1331597328009 to 1331682853011, 1524036253006 to 1524059653012, 1524068653006 to 1524074053011, and 1652783023007 to 1653177223016 all show distances of 0 and do not render in JHelioviewer. STEREO B has the same from 1331597369222 to 1331682894224.

**Comment by dgarciabriseno** [comment]
Hi, sorry for taking so long to look into this.
I confirmed that STEREO after time 1331597328009 (I used 2012-03-13T08:22:00 - 2012-03-14T08:22:00) don't render in JHelioviewer, they do render in helioviewer.org, most likely something in the metadata.

I did get the same error on 2011/12/25 06:39Z, I can look into that.

@wafels is out this week, but next week I'll have him take a look at the missing images.

@Jesterface23 For the spreadsheet you attached, are the dates for any specific source? (STEREO A/B/SOHO?).
As a side note, we don't have a lot of bandwidth for repairing images, it's a pretty involved process so it will take a while to get around to.

**Comment by dgarciabriseno** [comment]
That patch fixes the `Duplicated time stamps: ` but unfortunately jpx files are cached and I can't easily pick and choose which ones to clear, so it's possible you hit a cached jpx that has this error. If this happens, just change the start time or end time by 1 second and it'll make a new jpx. Cache is based on start/end times.

**Comment by dgarciabriseno** [comment]
@bogdanni Can you look into why some images won't render in JHelioviewer?
Query STEREO-A COR2 on dates 2012-03-12T04:00:00 - 2012-03-14T00:00:00
The images where the rendering breaks is here:
Rendered Correctly - https://helioviewer.org/jp2/COR2-A/2012/03/12/white-light/2012_03_12__23_39_00_009__STEREO-A_SECCHI_COR2_white-light.jp2 
Not Rendered - https://helioviewer.org/jp2/COR2-A/2012/03/13/white-light/2012_03_13__00_08_15_008__STEREO-A_SECCHI_COR2_white-light.jp2

**Comment by bogdanni** [comment]
The best would be regenerate those JP2 files if there are updated FITS files with correct metadata.

**Comment by bogdanni** [comment]
I've committed a change to log those bad DSUN_OBS values, will be in version 4.4 or later. The log can be consulted via menu Help>Show Log...

**Comment by dgarciabriseno** [comment]
Yep, I am using an older version of JHelioviewer, I need to update. Thanks for clearing that up.

**Comment by Jesterface23** [comment]
@dgarciabriseno 
They were a mix of ROB, IAS, or GSFC, depending on which had the larger file size over the span of 5 days.

Most of SOHO C3's imagery issues would be comparing the below to the Helioviewer imagery,
https://iswa.ccmc.gsfc.nasa.gov/IswaSystemWebApp/index.jsp?i_1=15&l_1=40&t_1=270&w_1=500&h_1=500&s_1=2013-10-10%2022:06:00.0_1_75_3

Given the process is going to take a while, one thing I've wanted to do is learn and figure out how to make the image files.

**Comment by dgarciabriseno** [comment]
Thanks, I see the dark spots that seem like they shouldn't be there. I agree these seem like they need to be reprocessed. @wafels has more info on how they're processed. I haven't worked directly with processing images, I just know they're processed with IDL which requires a license to run.
